typescript/phoenix-sdk/src/token.ts

```diff
@@ -1,25 +1,34 @@
 import { TokenParams } from "./types";
 
 export class Token {
-  name: string;
-  symbol: string;
-  logoUri: string;
-  data: TokenParams;
+  name?: string;
```

Comment:
why did we need to make these optional?

Reply:
see my comment above on objects with all undefined properties, these should be required